ui/packages/consul-ui/app/components/custom-element/README.mdx

```diff
@@ -1,87 +0,0 @@
-# CustomElement
```

Comment:
I saw the shadow template component was using an icon I removed in the documentation, then I checked where it and it's related components (shadow-host/custom-element/disclosure-card/etc.) were being used and saw that they weren't being used at all. I decided to remove them all. If we really want them back we can reintroduce them through git when they need to be used.

![snap](https://media4.giphy.com/media/LOoaJ2lbqmduxOaZpS/giphy.gif)